step-templates/RavenDB - Create Database

```diff
@@ -0,0 +1,103 @@
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "ravenDatabaseURL",
+      "Label": "URL of the RavenDB",
+      "HelpText": "URL of the RavenDB, where you would like the database to be created.  \n\nFor example **http://localhost:8080/**",
+      "DefaultValue": "http://localhost:8080/",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenDatabaseName",
+      "Label": "Name of the Database",
+      "HelpText": "The name the database will be called. Make sure that name is not already being used.",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenActiveBundles",
+      "Label": "Active Bundles",
+      "HelpText": "A settings option, Active Bundles refers bundles that RavenDB has (Replication;Versioning; etc)",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenStorageTypeName",
+      "Label": "Name of the Storage Type",
+      "HelpText": "What storage type to use",
+      "DefaultValue": "Esent",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Essent|Display Esent\nVoron|Display Voron"
```

Comment:
Should `Essent` be `Esent` and should the display values be `Display {storagetype}` i.e. `Display Esent` or should they be just `{storagetype}`?